common/http/src/retry.rs

```diff
@@ -154,6 +155,9 @@ impl Retry {
             }
         }
 
+        self.disk_used.fetch_sub(file_size, SeqCst);
+        Metrics::retry().report_storage_used(self.disk_used.load(SeqCst));
```

Comment:
Minor: we could use the atomic result of `fetch_sub` instead of reloading it.

Reply:
Yes. Atomic ops are expensive, I was suggesting using a single one (minor improvement in perf, bad for readability) 🙂 

I don't feel strongly about, tho.

Reply:
Addressed in commit https://github.com/logdna/logdna-agent-v2/pull/279/commits/9d0881cda62a8c0e2b50802810b6ada6f20ff1e6.